001-core/privacy.php

```diff
@@ -177,14 +179,26 @@ function generate_personal_data_export_file( $request_id ) {
 	// We can't currently iterate through files in the Files Service so we need a way to query exports by date.
 	update_post_meta( $request_id, '_vip_export_generated_time', time() );
 
+	$local_archive_pathname = $archive_pathname;
+	// Hack: ZipArchive and PclZip don't support streams.
+	// So, let's force the path to use a local one in the temp dir, which will work.
+	// All other references (meta) will still use the correct stream URL. 
+	if ( 0 === strpos( $local_archive_pathname, 'vip://' ) ) {
+		$local_archive_pathname = get_temp_dir() . substr( $archive_pathname, 6 );
+
+		// Create the folder path
+		$local_archive_dirname = dirname( $local_archive_pathname );
+		wp_mkdir_p( $local_archive_dirname );
```

Comment:
Should we add a check to ensure the directory path was properly created?

Reply:
Yep, good catch 👍

Reply:
Good catch; done in 6c5c78a62070d020d994bb2ace17ce2fea5a10d7